apps/explorer/src/app/routes/markets/market-page.tsx

```diff
@@ -0,0 +1,75 @@
+import { t, useDataProvider, useYesterday } from '@vegaprotocol/react-helpers';
+import { AsyncRenderer, Button } from '@vegaprotocol/ui-toolkit';
+import { useMemo, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { MarketDetails } from '../../components/markets/market-details';
+import { RouteTitle } from '../../components/route-title';
+import { useScrollToLocation } from '../../hooks/scroll-to-location';
+import { useDocumentTitle } from '../../hooks/use-document-title';
+import compact from 'lodash/compact';
+import { Interval } from '@vegaprotocol/types';
+import { JsonViewerDialog } from '../../components/dialogs/json-viewer-dialog';
+import { marketInfoDataProvider } from '@vegaprotocol/market-info';
+
+export const MarketPage = () => {
+  useScrollToLocation();
+
+  const { marketId } = useParams<{ marketId: string }>();
+
+  const yesterday = useYesterday();
+  const yTimestamp = useMemo(() => {
+    return new Date(yesterday).toISOString();
+  }, [yesterday]);
+  const variables = useMemo(
+    () => ({
+      marketId,
+      since: yTimestamp,
+      interval: Interval.INTERVAL_I1H,
+    }),
+    [marketId, yTimestamp]
+  );
```

Comment:
**question (non-blocking)**:  I feel silly asking, but it's not clear what this calculation is for. I can tell we want the data provider to give us... something since yesterday. I guess it's a 24 hour average  or something, but it all feels quite obscure. 

Maybe some variable naming would clear it up?

Non-blocking because me not understanding is not a problem in itself 😄